Corgibytes.Freshli.Cli/Functionality/Cache.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+using System.IO;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+using ServiceStack;
+
+namespace Corgibytes.Freshli.Cli.Functionality
+{
+    [Index(nameof(Key), IsUnique = true)]
+    public class CachedProperty
+    {
+        public int Id { get; set; }
+        public string Key { get; set; }
+        public string Value { get; set; }
+    }
+
+    class CacheContext : DbContext
+    {
+        public static readonly string cacheDirEnvVariable = "FRESHLI_CACHE_DIR";
+        public static readonly DirectoryInfo cacheDirDefault = new DirectoryInfo(Environment.GetEnvironmentVariable("HOME") + "/.freshli");
+        public static readonly string cacheDbName = "freshli.db";
```

Comment:
In C#, the naming convention is for public fields, properties, methods, etc... to be `PascalCased`. Also, const is a better fit than `readonly` in when the value is a constant

```suggestion
        public static const string CacheDirEnvVariable = "FRESHLI_CACHE_DIR";
        public static readonly DirectoryInfo CacheDirDefault = new DirectoryInfo(Environment.GetEnvironmentVariable("HOME") + "/.freshli");
        public static const string CacheDbName = "freshli.db";
```